AdobeAcrobatPro/AdobeAcrobatProUpdateInfoProvider.py

```diff
@@ -131,7 +127,7 @@ def get_manifest_data(self, manifest_plist_url):
         """Get manifest(plist) data from a url"""
         manifest_plist_response = self.get_url_response(manifest_plist_url)
         try:
-            manifest_data = readPlistFromString(manifest_plist_response)
+            manifest_data = readPlistFromString(manifest_plist_response.encode())
```

Comment:
Does this work for both plistlib.readPlistFromString (Py2) _and_ readPlistFromBytes (Py3)?

Reply:
Good catch. I'll try to use `readPlistFromBytes` for both.

Reply:
`readPlistFromBytes` not present in python2 plistlib. `readPlistFromString(manifest_plist_response.encode())` seems to work in the python2 code. However I am testing this on master + this applied change only since `py2-to-3` containes python2 breaking changes. Could you remove these changes (f-strings for instance) since you intend to keep python2 comatibility?

Reply:
`encode()` doesn't appear to have any effect on string object in Python 2.